pkg/sac/allow_fixed_scopes_checker_core.go

```diff
@@ -52,6 +53,81 @@ func (c allowFixedScopesCheckerCore) SubScopeChecker(key ScopeKey) ScopeCheckerC
 }
 
 func (c allowFixedScopesCheckerCore) EffectiveAccessScope(resource permissions.ResourceWithAccess) (*effectiveaccessscope.ScopeTree, error) {
-	// TODO(ROX-9537): Implement it
-	panic("Implement me: ROX-9537")
+	if len(c) == 0 {
+		return effectiveaccessscope.UnrestrictedEffectiveAccessScope(), nil
```

Comment:
Why not Deny All by default?

Reply:
The TryAllow function (lines 34-39) returns `Allow` when the list is empty.
The current logic (Unrestricted) ensures a consistent behaviour.